README.md

```diff
@@ -257,7 +257,29 @@ only supports manual instrumentation using the `opentelemetry-api` version with
 number as the Java agent you are using. Starting with 1.0.0, the Java agent will start supporting
 multiple (1.0.0+) versions of `opentelemetry-api`.
 
-You can use the OpenTelemetry `getTracer` or the `@WithSpan` annotation to
+You'll need to add a dependency on the `opentelemetry-api` library to get started.
+
+### Maven
+
+```xml
+  <dependencies>
+    <dependency>
+      <groupId>io.opentelemetry</groupId>
+      <artifactId>opentelemetry-api</artifactId>
+      <version>0.7.0</version>
+    </dependency>
+  </dependencies>
+```
+
+### Gradle
+
+```groovy
+dependencies {
+	compile('io.opentelemetry:opentelemetry-api:0.7.0')
+}
+```
+
+Now you can use the OpenTelemetry `getTracer` or the `@WithSpan` annotation to
```

Comment:
Thanks @johnbley, this definitely seems like a nice improvement! The only point I have is this gets us `getTracer` but for `@WithSpan` I believe a different annotations dependency is required.

https://github.com/open-telemetry/opentelemetry-java/tree/master/extensions/auto_annotations

Do you think you can add that too? Or we can follow up in a separate PR, no worries.

Reply:
Nice catch; I'll add that now.